# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/fetch/FetchStreamUtils.h
# Commit: c0cb3c17f246
# Full Hash: c0cb3c17f2461a19203f3b295a6c56b417483b2a
# Author: Eden Chuang <echuang@mozilla.com>
# Date: 2023-01-12 21:30:33
# Regressor Bug: 1351231
# File Overlap Count: 2
# Description:
#   Bug 1351231 - FetchService integration for PFetch. r=dom-worker-reviewers,jesup
#   
#   Depends on D142436
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D142437
# ==============================================================================

diff -r 84162d09eef8 -r c0cb3c17f246 dom/fetch/FetchStreamUtils.h
--- a/dom/fetch/FetchStreamUtils.h	Thu Jan 12 15:09:45 2023 +0000
+++ b/dom/fetch/FetchStreamUtils.h	Thu Jan 12 15:09:45 2023 +0000
@@ -40,7 +40,8 @@
 // process. Can only be called in the parent process.
 ParentToChildStream ToParentToChildStream(
     const NotNull<nsCOMPtr<nsIInputStream>>& aStream, int64_t aStreamSize,
-    NotNull<mozilla::ipc::PBackgroundParent*> aBackgroundParent);
+    NotNull<mozilla::ipc::PBackgroundParent*> aBackgroundParent,
+    bool aSerializeAsLazy = true);
 
 // Convert a ParentToParentStream to a ParentToChildStream. Can only be called
 // in the parent process.